code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm

```diff
@@ -570,14 +570,14 @@
 	data["components"] = stored_components.Copy()
 	var/list/temp_data = list("<font color=#B18B25>")
 	for(var/i in data["components"]) //display the slab's components
-		temp_data += "<font color=[get_component_color_bright(i)]>[get_component_acronym(i)] <b>[data["components"][i]]</b></font>"
+		temp_data += "<font color=[get_component_color_bright(i)]>[get_component_icon(i)] <b>[data["components"][i]]</b></font>"
 		if(i != HIEROPHANT_ANSIBLE)
 			temp_data += " "
 		else
-			temp_data += " ("
+			temp_data += " <br>("
```

Comment:
It's to separate the component count and the "with global cache" count; it doesn't seem to have any adverse effects, as you can see it in the screenshot. Is there something I missed that I should be accounting for?